raft.go

```diff
@@ -488,7 +607,7 @@ func (r *Raft) Stats() map[string]string {
 		"fsm_pending":         toString(uint64(len(r.fsmCommitCh))),
 		"last_snapshot_index": toString(lastSnapIndex),
 		"last_snapshot_term":  toString(lastSnapTerm),
-		"num_peers":           toString(uint64(len(r.peers))),
+		"num_servers":         toString(uint64(len(r.configurations.latest.Servers))),
```

Comment:
A list of servers doesn't have all the information in this new world. What is this function used for?